smallrye-reactive-messaging-kafka/pom.xml

```diff
@@ -113,6 +113,11 @@
       <version>1.0.3</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-log4j12</artifactId>
+      <version>1.7.32</version>
```

Comment:
Should we add "optional"?

Reply:
Yes, definitely it should be "test" scope. I had it on the kafka-companion branch, but better push it here  and I can rebase the other one.